var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -14,7 +14,9 @@ class Paraview(CMakePackage):
     homepage = 'http://www.paraview.org'
     url      = "http://www.paraview.org/files/v5.3/ParaView-v5.3.0.tar.gz"
     _urlfmt  = 'http://www.paraview.org/files/v{0}/ParaView-v{1}{2}.tar.gz'
+    git      = "https://gitlab.kitware.com/vtk/vtk.git"
 
+    version('develop', branch='master')
```

Comment:
I think master should be listed as master.

Reply:
There isn't really consistency across spack on versioning of development branches.  Some packages use the repo branch name while others use develop regardless, representing the generic concept of a development version and not tying it to the underlying source control implementation.  The second seems to be more common amongst spack packages and is how most of the viz and io packages are choosing to represent a development version in spack.

Reply:
The usage of `develop` instead of `master` is so prevalent because it was a workaround before `master` meant "newer than stable releases". Now that that's fixed, I would encourage users to name versions according to the actual branch name. I've been meaning to do a Spack-wide update of develop -> master but haven't had the time.

Reply:
Oh that was finally fixed? That's great. I was going to mention that I wish this practice would go away.

Reply:
Yep, see #1983

Reply:
Since it's the most common practice now to use 'develop' regardless of underlying branch name or source control mechanism, I wouldn't assert this to be incorrect until an official policy codified in the documentation conveys otherwise.